src/core/validation/CustomValidationRules.ts

```diff
@@ -148,6 +148,15 @@ export class CustomValidationRules {
       },
       message: `${i18n.t('error_new_password_format')}`,
     })
+
+    extend('delHtmlTag', {
```

Comment:
could you please rename to something without "del" (this function does not delete anything)

would be nice if you can add come tests

Reply:
rename to common `stripTags`

Reply:
added tests